src/CommandHandler.ts

```diff
@@ -241,7 +241,16 @@ export namespace CommandHandler {
 		await client.login(options.token);
 		prefixes.push(`<@${client?.user?.id}> `);
 		prefixes.push(`<@!${client?.user?.id}> `);
-		owners.push((await client.fetchApplication()).owner?.id ?? '');
+		const appOwner = (await client.fetchApplication()).owner;
+		if (appOwner) {
+			if (appOwner instanceof Team) {
+				for (const member of appOwner.members.array()) {
+					owners.push(member.id);
+				}
```

Comment:
Can be replaced with `owners.push(...appOwner.members.array().map(m => m.id));`.

Reply:
Didn't know this thing, I update this